### DISCUSSION TITLE
ui5-combobox without hide-arrow

### DISCUSSION THREAD

**Discussion by ppithon3** [UI5/webcomponents#6234] [General]
Hello,

ui5-combobox use a ui5-responsive-popover for display item list

I want to display arrow in the popup

![image](https://user-images.githubusercontent.com/33153391/210084091-404e7b71-ea35-406f-8ae8-0ecb01c704b8.png)

but hide-arrow is forced in template ComboboxPopover.hbs : 

```
<ui5-responsive-popover hide-arrow 

```

Any ideas ?

**Comment by petyabegovska** [comment]
Hi @ppithon3,

Is the request still valid? If yes, you can transfer it as a new feature request ('create issue from discussion').

Best regards,
Petya
